src/app/content/highlights/components/DisplayNote.tsx

```diff
@@ -40,8 +40,8 @@ interface Props {
 }
 
 // tslint:disable-next-line:variable-name
-const DisplayNote = React.forwardRef<HTMLElement, Props>((
-  {note, isFocused, onBlur, onEdit, onRemove, onFocus, onHeightChange, className}: Props,
+const DisplayNote = React.forwardRef<HTMLElement, DisplayNoteProps>((
+  {note, isFocused, onBlur, onEdit, onRemove, onFocus, onHeightChange, className},
```

Comment:
I've checked it again and I've noticed that the generic type adds `children` to the supplied type. This is my bad, sorry.

Reply:
I think this is actually better, even if we do not use `children`
btw. I've applied your suggestions